src/lib/ChargeStation/eventHandlers/ocpp-16/sicharge/handle-data-transfer.ts

```diff
@@ -0,0 +1,16 @@
+import { ChargeStationEventHandler } from 'lib/ChargeStation/eventHandlers';
+import { DataTransferRequest } from 'schemas/ocpp/1.6/DataTransfer';
+
+const handleDataTransfer: ChargeStationEventHandler<DataTransferRequest> = ({
+  chargepoint,
+  callMessageId,
+  callMessageBody,
+}) => {
+  const { vendorId } = callMessageBody;
+  const status = vendorId === 'AdhocPayment' ? 'Accepted' : 'UnknownVendorId';
+  chargepoint.writeCallResult(callMessageId, {
+    status,
+  });
```

Comment:
This is just to prevent `UnknownVendorId` being returned for now. In the future we can perhaps do something with the payloads